src/execution_plan/ops/op_argument.c

```
@@ -7,63 +7,130 @@
+	ASSERT(arg != NULL);
+
+	// free current 'r' and '_r'
+	if(arg->r != NULL) {
+		OpBase_DeleteRecord(&arg->r);
+	}
+
+	if(arg->_r != NULL) {
+		OpBase_DeleteRecord(&arg->_r);
+	}
+
+	// set record
```

Cppcheck (2.10-2)

[error] Common realloc mistake

(memleakOnRealloc)